### PR TITLE
doc: Add CLI reference docs part2 and rename the reference docs

### PR DESCRIPTION
**Pull request by FeynmanZhou**
fix https://github.com/oras-project/oras-www/issues/59. These CLI reference docs are generated using Cobra. I just polished the format and layout manually. 
We are looking to automate this process in the future.

### CODE REVIEW

**Comment by TerryHowe** [comment]
Are these pages generated?

File: docs/CLI_Reference/oras_push.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# oras push
+
+Push files to remote registry.
+
+```bash
+oras push [flags] <name>[:<tag>[,<tag>][...]] <file>[:<type>] [...]
+```
+
+## Examples
+
```
Comment (TerryHowe) [inline docs/CLI_Reference/oras_push.md:10]: These examples lack `oras push`

File: docs/CLI_Reference/oras_repo_tags.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## Options
+
+```
+      --ca-file string              server certificate authority file for the remote registry
+  -d, --debug                       debug mode
+      --exclude-digest-tags         exclude all digest-like tags such as 'sha256-aaaa...'
+  -h, --help                        help for tags
+      --insecure                    allow connections to SSL registry without certs
+      --last last                   start after the tag specified by last
+      --oci-layout                  Set target as an OCI image layout.
```
Comment (TerryHowe) [inline docs/CLI_Reference/oras_repo_tags.md:62]: This option is a capitalized and with a period at the end for some reason. I noticed it is like that for other commands as well.

File: docs/community.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
-Want to reach the ORAS community and developers?
-We're very interested in feedback and contributions for other artifacts.
+Welcome to the ORAS Project community.
+
+- [Join us](https://slack.cncf.io/) at [CNCF Slack](https://cloud-native.slack.com) under the **#oras** channel
+- Join the weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
```
Comment (Mr-DG-Wick) [inline docs/community.md:4]: shouldn't it be bi-weekly?
```suggestion
- Join the bi-weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
```

**Comment by FeynmanZhou** [comment]
> Are these pages generated?

Yes, it was generated by Cobra. I just polished the format and layout. See https://github.com/oras-project/oras-www/issues/59

We are looking to automate this process in the future.

File: docs/CLI_Reference/oras_repo_ls.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Examples
+
+Example - List the repositories under the registry:
+
+```bash
+  oras repo ls localhost:5000
+```
+
```
Comment (qweeah) [inline docs/CLI_Reference/oras_repo_ls.md:16]: There is a new example:
~~~suggestion
Example - List the repositories under a namespace in the registry:
```bash
  oras repo ls localhost:5000/example-namespace
```
~~~

**Review by qweeah — CHANGES_REQUESTED** [review]
It's better 1) regenerate all the docs based on 1.0.0; 2) replace `options` section with new docs' options section.

File: docs/CLI_Reference/oras_push.md (hunk 1 above)
Comment (FeynmanZhou) [reply docs/CLI_Reference/oras_push.md:10]: Fixed

File: docs/CLI_Reference/oras_repo_tags.md (hunk 2 above)
Comment (FeynmanZhou) [reply docs/CLI_Reference/oras_repo_tags.md]: Fixed

File: docs/CLI_Reference/oras_login.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  -d, --debug                       debug mode
+  -h, --help                        help for login
+      --insecure                    allow connections to SSL registry without certs
+  -p, --password string             registry password or identity token
+      --password-stdin              read password or identity token from stdin
+      --plain-http                  allow insecure connections to registry without SSL check
+      --registry-config path        path of the authentication file
+      --resolve host:port:address   customized DNS formatted in host:port:address
+  -u, --username string             registry username
+  -v, --verbose                     verbose output
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply docs/CLI_Reference/oras_login.md]: Updated all options section

File: docs/CLI_Reference/oras_repo_ls.md (hunk 4 above)
Comment (FeynmanZhou) [reply docs/CLI_Reference/oras_repo_ls.md:22]: Added this example

**Review by TerryHowe — APPROVED** [review]
LGTM

File: docs/CLI_Reference/oras_attach.md
Comment (qweeah) [inline docs/CLI_Reference/oras_attach.md:1]: Options section is not updated to 1.0.0.

File: docs/CLI_Reference/oras_blob_delete.md
Comment (qweeah) [inline docs/CLI_Reference/oras_blob_delete.md:1]: Options section is not updated to 1.0.0.

File: docs/CLI_Reference/index.md [hunk 6]
```diff
@@ -31,13 +31,13 @@ Use "oras [command] --help" for more information about a command.
...
+* [oras attach](oras_attach.md)	 - Attach files to an existing artifact
 * oras blob - Blob operations
-    * [oras blob delete](1_oras_blob_delete.md)	 - Delete a blob from a remote registry
-    * [oras blob fetch](2_oras_blob_fetch.md)	 - Fetch a blob from a remote registry
-    * [oras blob push](3_oras_blob_push.md)	 - Push a blob to a remote registry
-    * [oras copy](4_oras_copy.md)	 - Copy artifacts from one target to another
-* [oras discover](5_oras_discover.md)	 - Discover referrers of a manifest in the remote registry
+    * [oras blob delete](oras_blob_delete.md)	 - Delete a blob from a remote registry
+    * [oras blob fetch](oras_blob_fetch.md)	 - Fetch a blob from a remote registry
+    * [oras blob push](oras_blob_push.md)	 - Push a blob to a remote registry
```
Comment (qweeah) [inline docs/CLI_Reference/index.md:38]: I found those descriptions are not accurate, e.g. `blob fetch` and `blob push` can be used to operate OCI layouts. Will create a PR to update the doc in ORAS CLI, for this PR I am okay to merge it this way.

File: docs/CLI_Reference/oras_blob_fetch.md
Comment (qweeah) [inline docs/CLI_Reference/oras_blob_fetch.md:1]: Options section is not updated to 1.0.0.

File: docs/CLI_Reference/oras_blob_push.md
Comment (qweeah) [inline docs/CLI_Reference/oras_blob_push.md:1]: Options section is not updated to 1.0.0.

File: docs/CLI_Reference/oras_copy.md
Comment (qweeah) [inline docs/CLI_Reference/oras_copy.md:1]: Options section is not updated to 1.0.0.

File: docs/CLI_Reference/oras_discover.md
Comment (qweeah) [inline docs/CLI_Reference/oras_discover.md:1]: Options section is not updated to 1.0.0.

**Review by qweeah — CHANGES_REQUESTED** [review]

File: docs/CLI_Reference/oras_blob_fetch.md
Comment (qweeah) [inline docs/CLI_Reference/oras_blob_fetch.md:1]: Options are not updated for `oras blob fetch`.

**Review by qweeah — CHANGES_REQUESTED** [review]

File: docs/CLI_Reference/oras_blob_fetch.md
Comment (FeynmanZhou) [reply docs/CLI_Reference/oras_blob_fetch.md:1]: Good catch. Updated

**Review by qweeah — APPROVED** [review]
LGTM but IANAM

**Pull request merged**